### ISSUE TITLE
Show Privacy Policy and Terms modal for upgraded users

### ISSUE THREAD

**Issue by pavloburykh** [status-im/status-react#21113]
When users upgrade from V1 to V2 app they need to accept updated Privacy Policy and Terms. 

For those purposes we need to show Privacy Policy modal after upgrade. 

**Designs**

https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=13865-45279&t=YzMcbvPYvKIJ31NR-1

![image](https://github.com/user-attachments/assets/812c0df2-e6aa-45e3-ad38-9126b30e27ca)

**Comment by churik** [comment]
it is done, thank you @ilmotta

**Comment by ilmotta** [comment]
@churik, I'll need to reopen this issue because it has to be merged into develop branches in status-go and mobile, and we only merged in release branches. And while doing that, I'll have to adapt the solution in status-go a little bit and ask for QA again. The solution I did was acceptable, but we agreed in PR reviews that I would improve in a follow-up.

Will try to get implementation done until end of week.
